Userland/Libraries/LibGfx/ImageFormats/GIFLoader.cpp

```diff
@@ -14,7 +14,7 @@
 #include <AK/Memory.h>
 #include <AK/MemoryStream.h>
 #include <AK/Try.h>
-#include <LibCompress/LZWDecoder.h>
+#include <LibCompress/LZW.h>
```

Comment:
This will match the casing of the other files of `LibCompress` but not other files from the system :smiley: (As an example, we have `JPEGLoader.cpp` not `JpegLoader.cpp`, `CSS` not `Css`.) I guess it makes more sense to respect the local law (and I will do when I have time to work on that PR), but I would actually rather rename every other files.

Reply:
> but I would actually rather rename every other files

I got curious and checked, looks like we might have a wilder mix throughout the system than I expected. I'd still argue to keep LibCompress consistent for now, everything else we should probably only do after documenting said cases in our coding style document.